src/test/java/ru/testtask/converter/ProjectDTOConverterTest.java

```diff
@@ -0,0 +1,103 @@
+package ru.testtask.converter;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import ru.testtask.converter.ProjectDTOConverter;
+import ru.testtask.dto.AttrDTO;
+import ru.testtask.dto.CreateProjectDTO;
+import ru.testtask.dto.ProjectDTO;
+import ru.testtask.model.Attribute;
+import ru.testtask.model.Project;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+
+@RunWith(MockitoJUnitRunner.class)
+public class ProjectDTOConverterTest {
+
+    private static ProjectDTOConverter projectDTOConverter;
+
+    private static Project project;
+    private static ProjectDTO projectDTO;
+    private static CreateProjectDTO createProjectDTO;
+
+    @Before
+    public void buildTestData(){
+        project = Project.builder().id("a").name("prj").attributes(new ArrayList<Attribute>())
```

Comment:
It is better to add some elements to `attributes` and `attrs` and check that this data is correctly mapped to the corresponding field.